website/pages/en/developing/supported-networks.mdx

```diff
@@ -31,13 +31,13 @@ import { getSupportedNetworks } from '@/src/getSupportedNetworks'
 
 The hosted service relies on the stability and reliability of the underlying technologies, namely the provided JSON RPC endpoints.
 
-You will **not be able** to publish a subgraph that indexes a non-mainnet network to the decentralized Graph Network in [Subgraph Studio](/deploying/subgraph-studio).
-
 Ropsten, Rinkeby and Kovan are being deprecated. Read more on the [Ethereum Foundation Blog](https://blog.ethereum.org/2022/06/21/testnet-deprecation). As of Feb 25th 2023, Ropsten, Rinkeby and Kovan are no longer supported by the Hosted Service. Goerli will be maintained by client developers post-merge, and is also supported by the Hosted Service. Developers who currently use Ropsten, Rinkeby or Kovan as their staging/testing environment are encouraged to migrate to Goerli.
 
 Subgraphs indexing Gnosis Chain can now be deployed with the `gnosis` network identifier. `xdai` is still supported for existing hosted service subgraphs.
 
-For a full list of which features are supported on the decentralized network, see [this page](https://github.com/graphprotocol/indexer/blob/main/docs/networks/mainnet.md#feature-support).
+For a full list of which features are supported on the decentralized network, see [this page](https://github.com/graphprotocol/indexer/blob/main/docs/feature-support-matrix.md).
+
+Substreams-powered subgraphs indexing mainnet Ethereum are supported on the Subgraph Studio and decentralized network.
```

Comment:
```suggestion
Substreams-powered subgraphs indexing `mainnet` Ethereum are supported on the Subgraph Studio and decentralized network.
```
(just to be consistent with [above](https://github.com/graphprotocol/docs/pull/446/files#diff-12bc548e5b4b5023c1fd7fea87b27b76c03276b08ca13c202f5b2908e66c3eccR162), but maybe we want to update above instead?)